miqa/core/rest/frame.py

```diff
@@ -125,13 +125,21 @@ def create(self, request, *args, **kwargs):
             scan.save()
         elif 'scan' in serializer.data:
             scan = Scan.objects.get(id=serializer.data['scan'])
+            if not scan:
+                raise APIException(
+                    'Could not create new Frame.'
+                )
             if not get_perms(request.user, scan.experiment.project):
                 Response(status=status.HTTP_403_FORBIDDEN)
         else:
             raise APIException(
                 'Provide either a parent scan or grandparent experiment for this frame.'
             )
 
+        if not scan:
+            raise APIException(
+                'Could not create new Frame.'
```

Comment:
Same here

Reply:
Changed in [d914ecf](https://github.com/OpenImaging/miqa/pull/634/commits/d914ecf3566871d90ced550a4722dacf42d057b9)